<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-30T16:50:57 IST" name="Default suite" finished-at="2022-09-30T16:51:05 IST" duration-ms="8487">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationTest.Standalo()[pri:0, instance:Framework_3.Nwpro.ErrorValidationTest@242b836]" name="Standalo" class="Framework_3.Nwpro.ErrorValidationTest"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2022-09-30T16:50:57 IST" name="Default test" finished-at="2022-09-30T16:51:05 IST" duration-ms="8487">
      <class name="Framework_3.Nwpro.ErrorValidationTest">
        <test-method signature="Standalo()[pri:0, instance:Framework_3.Nwpro.ErrorValidationTest@242b836]" started-at="2022-09-30T16:50:57 IST" name="Standalo" finished-at="2022-09-30T16:51:05 IST" duration-ms="8401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Standalo -->
      </class> <!-- Framework_3.Nwpro.ErrorValidationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
